modules/mssqlserver/src/main/java/org/testcontainers/containers/MSSQLR2DBCDatabaseContainerProvider.java

```diff
@@ -21,6 +21,7 @@ public boolean supports(ConnectionFactoryOptions options) {
 
     @Override
     public R2DBCDatabaseContainer createContainer(ConnectionFactoryOptions options) {
+        // TODO work out how best to do this if these constants become private
```

Comment:
A TODO for a point in the near future. This has a lot to do with mandatory bring-your-own-image in R2DBC and JDBC URLs as discussed in Slack (@bsideup)

Reply:
Yeah, that's probably going to be the answer. This isn't something to worry about too much for now, anyway.